shared/overview/2.x.md

```diff
@@ -42,15 +42,15 @@ To persist logs on the device, enable persistent logging via the [fleet configur
 
 Both development and production versions of {{ $names.os.lower }} allow the setting of a custom [hostname][config-json-hostname] via `config.json`, by setting `"hostname": "my-new-hostname"`. Your device will then broadcast (via Avahi) on the network as `my-new-hostname.local`. If you don't set a custom hostname, the device will default to `<short-UUID>.local`. You can also set a custom hostname via the [Supervisor API][supervisor-api] on device.
 
-On production images, nothing is written to tty1, on boot up you should only see the {{ $names.company.lower }} logo on the HDMI screen, and this will persist until your application code takes over the framebuffer. If you would like to replace the {{ $names.company.lower }} logo with your own custom splash logo, then you will need to replace splash/resin-logo.png file that you will find in the [first partition][parition] of the image (boot partition or `resin-boot`) with your own logo.
+On production images, nothing is written to tty1, on boot up you should only see the {{ $names.company.lower }} logo on the HDMI screen, and this will persist until your application code takes over the framebuffer. If you would like to replace the {{ $names.company.lower }} logo with your own custom splash logo, then you will need to replace the `splash/resin-logo.png` file that you will find in the [first partition][parition] of the image (boot partition or `resin-boot`) with your own logo.
```

Comment:
Is "the HDMI screen" overly specific, given the wide range of devices supported by Balena? :man_shrugging: